packages/client3/components/design-system/divider/divider.tsx

```diff
@@ -0,0 +1,13 @@
+import clsx from "clsx";
+
+type Props = {
+  className?: string;
+};
+
+export function Divider({ className }: Props) {
+  return (
+    <div className={clsx("relative flex items-center py-5", className)}>
+      <div className="flex-grow border-t border-dark-90"></div>
+    </div>
+  );
+}
```

Comment:
Sorry does this work instead?

https://tailwindcss.com/docs/divide-width

Reply:
Yeah, I have tried it but then it only works when a divider is required for each grid component